microsoft-outlook/latest/install/index.md

```diff
@@ -124,7 +122,76 @@ Ensure that you have applied the Alfresco Outlook Server AMP files ([see Install
+
+
+**TODO:**
+* *Add more info about the Docker container and WAR file use cases?*
+* *Separate "Install AMPs" + "Install T-Engine on Tomcat" from "Install using Docker"?*
+* *Add "Containerized deployment" sub-section?*
+ 
+
+### Install T-Engine on Tomcat
+
+If you wish to use an existing Tomcat application server, you can use the WAR bundle to install the Outlook Integration T-Engine.
+
+1. Install the latest version of [Tomcat 8.5](https://tomcat.apache.org/download-80.cgi){:target="_blank"} to run as a service.
+
+    See the [Tomcat documentation](https://tomcat.apache.org/tomcat-8.5-doc/index.html){:target="_blank"} for more information about configuring and using a Tomcat server.
+
+    For information about securing Tomcat, see [Tomcat security considerations](https://tomcat.apache.org/tomcat-8.5-doc/security-howto.html).
+
+2. Copy the WAR file into your `<TOMCAT_HOME>/webapps` folder.
+
+    The WAR (`transform-outlook-webapp-1.0.0.war`) is included in the distribution zip.
+
+3. Start the Tomcat service.
+
+4. Test that the T-Engine is running:
+
+    1. Open your browser and access `http(s)://${SERVER}:{PORT}/transform-outlook` to load the T-Engine test page.
+
+    2. Add the following configuration property in `alfresco-global.properties`:
+
+        ```bash
+        localTransform.transform-outlook.url=http(s)://${SERVER}:{PORT}/transform-outlook
+        ```
+
+## Install using Docker Compose
+
+To deploy the Outlook Integration T-Engine as part of Content Services, you'll need to update your Docker Compose file for Content Services to include this T-Engine.
+
+> **Note:** While Docker Compose is often used for production deployments, the Docker Compose file provided is recommended for development and test environments only. Customers are expected to adapt this file to their own requirements, if they intend to use Docker Compose to deploy a production environment.
+
+1. Add Outlook Integration T-Engine container to your `docker-compose.yaml` file:
+
+    ```yaml
+    transform-outlook:
+        image: alfresco/transform-outlook:1.0.0-RELEASE
+        mem_limit: 2g
+        environment:
+            JAVA_OPTS: " -Xms256m -Xmx512m"
+            ACTIVEMQ_URL: "nio://activemq:61616"
+            ACTIVEMQ_USER: "admin"
+            ACTIVEMQ_PASSWORD: "admin"
+            FILE_STORE_URL: "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file"
+        ports:
+            - 8091:8090
+        links:
+            - activemq
+    ```
+
+2. Add the following `JAVA_OPTS` variable to the `alfresco` container:
+
+    ```yaml
+    -DlocalTransform.transform-outlook.url=http://transform-outlook:8090/
+    ```
+
+3. **TODO** - needs more info: notes / start up / shutdown / troubleshooting...
```

Comment:
Link to https://github.com/Alfresco/acs-packaging/blob/master/docs/creating-a-t-engine.md